core/templates/pages/classrooms-page/classrooms-page.component.html

```diff
@@ -0,0 +1,277 @@
+<div class="oppia-classrooms-page-container">
+  <div class="breadcrumbs-container">
+    <div class="classrooms-breadcrumbs">
+      <div class="desktop-breadcrumbs">
+        <a href="/">Home</a>
+        <i class="fa fa-angle-right"></i>
+        <span>Classrooms</span>
+      </div>
+      <div class="mobile-breadcrumbs">
+        <i class="fa fa-angle-left"></i>
+        <a href="/">Back to Home</a>
+      </div>
+    </div>
+  </div>
+
+  <div *ngIf="(DEV_MODE && classroomSummaries.length === 0)"
+       class="classroom-admin-banner"
+  >
+    <span>To create classrooms, visit the <a href="/classroom-admin">classroom admin</a> page.</span>
+  </div>
+
+  <div class="classrooms-page-content-container">
+    <div class="classrooms-page-content">
+      <h1 class="classrooms-heading">Oppia Classrooms</h1>
+      <div class="classrooms-main-content">
+        <img alt="Illustration image for classrooms"
+             class="classrooms-illustration-image"
+             src="/assets/images/classroom/classrooms-page-illustration.svg">
+        <div class="description-container">
+          <p class="classrooms-description">
+            {{ 'I18N_MULTIPLE_CLASSROOMS_PAGE_DESCRIPTION' | translate }}
+          </p>
+        </div>
+        <div class="classroom-tile-container">
+          <ng-container *ngFor="let classroomSummary of classroomSummaries">
+            <ng-container *ngIf="classroomSummary.is_published">
+              <oppia-classroom-summary-tile [classroomSummary]="classroomSummary"></oppia-classroom-summary-tile>
+            </ng-container>
+          </ng-container>
+          <ng-container *ngIf="haveAtleastOnePrivateClassroom">
+            <oppia-classroom-summary-tile [classroomSummary]="privateClassroomSummary"></oppia-classroom-summary-tile>
+          </ng-container>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="oppia-classrooms-search-container">
```

Comment:
Huh, where's this search container? I haven't seen it in the proof of changes

Reply:
It's this one. we have oppia-search-bar component which we show in almost all pages but we might show different text above it according to the page.

we need to update oppia-search-bar component too but it's not related to my change I guess.

Desktop
![Screenshot from 2024-06-22 23-38-04](https://github.com/oppia/oppia/assets/79896602/00c11a65-1ef1-43f3-95ab-8ba181d31cff)

Mobile
![Screenshot from 2024-06-22 23-38-21](https://github.com/oppia/oppia/assets/79896602/740078db-56c6-48f3-92ae-09e507f2493e)